web/src/pages/BookWizard/TabContentPage.tsx

```diff
@@ -34,6 +36,11 @@ export const TabContentPage: React.FC<TabConentPageProps> = ( {onContentChange,
     setVideo(currentContent.video);
   }, []);
 
+  useEffect( () => {
+    setBody(currentContent.body);
+    setVideo(currentContent.video);
+  }, [language]);
```

Comment:
I think currentContent would be the better dependency here right? Language probably works too but currentContent should work and makes more sense.

Reply:
if you do current content, react complains that the "maximum depth has been exceeded". current content change causes video and body to be changed,which caused the reac tab content map to be changed (in the parent), which updates the current content again. It loop infinitely